scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ChangeHttpConfiguration.java

```diff
@@ -350,11 +358,13 @@ private static void setCurrentConfiguration(String scheme, String port) {
             ChangeHttpConfiguration.port = isHttps ? WebProperties.WEB_HTTPS_PORT.getValueAsInt()
                                                    : WebProperties.WEB_HTTP_PORT.getValueAsInt();
         }
+        System.out.println("Applying configuration: scheme=" + (isHttps ? "https" : "http") + " hostname=" +
+                           (hostname != null ? hostname : "<unchanged>") + " port=" + ChangeHttpConfiguration.port);
     }
 
     private static String answerAndLog(String line) {
         if (isDebug) {
-            log(" -> Set " + line);
+            log("DEBUG:  -> Set " + line);
```

Comment:
maybe the debug is not needed for -> Set, not sure

Reply:
I meant the hole block could be a debug line 
maybe not a good idea

Reply:
up to you

Reply:
I kept this as it was